tensorflow/lite/interpreter.cc

```diff
@@ -246,6 +246,14 @@ TfLiteStatus Interpreter::SetExecutionPlan(const std::vector<int>& new_plan) {
 void Interpreter::UseNNAPI(bool enable) { primary_subgraph().UseNNAPI(enable); }
 
 void Interpreter::SetNumThreads(int num_threads) {
+  // num_threads should be greater than -1.
+  // User may use -1, to reset to default values.
+  if (num_threads < -1) {
```

Comment:
"-1" really indicates that "the #threads will be set to default values in TFLite interpreter, and the default value might be 1 or other values". Admittedly, the current default #threads used inside TFLite interpreter is just 1. Could you add a comment about this logic?

Reply:
@multiverse-tf : The default value changes from context to context, so we can not put those values here, as later if those changes, this info will become stale, so it needs some maintenance. I think that comment may not be required. Please let me know your opinion. Thanks!

Reply:
My comment is about to make the meaning of "-1" less confusing here.  Particularly, as for your comment "User may use -1, to reset to default values", at the first sight, I thought the default value is "-1". Also, you might want to add such comment to the header file where this function is declared.

Reply:
@multiverse-tf : Thanks for the clarity! May be i can add an example, like below:
Cpu Context Default Thread Count: X
if num_threads == -1:
    Context::max_threads = X

In this way it will be more representational than exposing the exact values.
Please let me know your opinion, Thanks!

Reply:
+1. It looks much more clear! Thx!